tests/scale_up_status_updates/scale_up_status_updates_suite_test.go

```diff
@@ -1,71 +0,0 @@
-package scale_up_status_updates
-
-import (
-	"fmt"
-	"testing"
-
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-
-	ginkgo_util "github.com/datastax/cass-operator/mage/ginkgo"
-	"github.com/datastax/cass-operator/mage/kubectl"
-)
-
-var (
-	testName     = "Scale up status updates"
-	namespace    = "test-scale-up-status-updates"
-	dcName       = "dc1"
-	dcYaml       = "../testdata/oss-two-rack-six-node-dc.yaml"
-	operatorYaml = "../testdata/operator.yaml"
-	dcResource   = fmt.Sprintf("CassandraDatacenter/%s", dcName)
-	dcLabel      = fmt.Sprintf("cassandra.datastax.com/datacenter=%s", dcName)
-	ns           = ginkgo_util.NewWrapper(testName, namespace)
-)
-
-func TestLifecycle(t *testing.T) {
-	AfterSuite(func() {
-		logPath := fmt.Sprintf("%s/aftersuite", ns.LogDir)
-		kubectl.DumpAllLogs(logPath).ExecV()
-		fmt.Printf("\n\tPost-run logs dumped at: %s\n\n", logPath)
-		ns.Terminate()
-	})
-
-	RegisterFailHandler(Fail)
-	RunSpecs(t, testName)
-}
-
-var _ = Describe(testName, func() {
-	Context("when in a new cluster", func() {
-		Specify("the operator can scale up a datacenter and does not upsert the super user until all nodes have been started", func() {
-			By("creating a namespace")
-			err := kubectl.CreateNamespace(namespace).ExecV()
-			Expect(err).ToNot(HaveOccurred())
-
-			step := "setting up cass-operator resources via helm chart"
-			ns.HelmInstall("../../charts/cass-operator-chart")
-
-			ns.WaitForOperatorReady()
-
-			step = "creating a datacenter resource with 2 racks/6 nodes"
-			k := kubectl.ApplyFiles(dcYaml)
-			ns.ExecAndLog(step, k)
-
-			ns.WaitForSuperUserUpserted(dcName, 600)
-
-			step = "checking that all nodes have been started"
-			nodeStatusesHostIds := ns.GetNodeStatusesHostIds(dcName)
-			Expect(len(nodeStatusesHostIds), 6)
```

Comment:
This test isn't really testing anything to do with scale up (as the test name suggests), it is only checking that we record all the host ids. I don't think this warrants its own dedicated test due to how simple it is, much like how most of the Condition statuses don't warrant their own test. I've moved this check into tests/oss_test_all_the_things/oss_test_all_the_things_suite_test.go.